### PR TITLE
Viewing an initaitive

### PR DESCRIPTION
**Pull request by Qianfinland**
the viewing-an-initative-test.js has error message similar as before, i write the test inside listing-initiatives-test.js

### CODE REVIEW

File: tests/acceptance/listing-initiatives-test.js [hunk 1]
```diff
@@ -19,4 +19,39 @@ test('Successfully', function(){
     expect(find('.initiative').first().text()).to.equal("Public health clinic");
     expect(find('.initiative').last().text()).to.equal("Public library upgrade");
   });
+  
+  // //test for viewing an initiative
+  // //test first initiative
+  visit('/initiatives').then(function() {
```
Comment (oliverbarnes) [inline tests/acceptance/listing-initiatives-test.js]: Hey @Qianfinland , clicking on an initiative on the list and viewing it should be its own test, as viewing an initiative is its own feature

File: tests/acceptance/listing-initiatives-test.js [hunk 2]
```diff
@@ -19,4 +19,39 @@ test('Successfully', function(){
...
+      expect(currentURL()).to.equal('/initiatives/1'); 
+      expect(find('.issue_title').text()).to.equal('Issue: What to do with the compensation money from the dam\'s impact?');
+      expect(find('.description').text()).to.equal('Allocate compensation money to create a local public health clinic');
+      expect( $("a:contains('Support this initiative')").length ).to.equal(1);
+      expect(find('.new_suggestion').text()).to.equal('Note: To make suggestions for an initiative, you must first support it');
+    });
+  });
+
+  // test second initiative
+  visit('/initiatives').then(function() {
```
Comment (oliverbarnes) [inline tests/acceptance/listing-initiatives-test.js]: you can test just one of them, btw. these don't add any more coverage

**Comment by oliverbarnes** [comment]
Thanks Qian, you'll have to get that error fixed before we can close this pull request. Probably just a sync problem. Interestingly the Travis CI build is passing (see here next to the merge pull request button)

**Comment by Qianfinland** [comment]
yeah, nice feedback, i will just test one and fix the error :)

**Comment by Qianfinland** [comment]
already fix the test error, just rename the test file, maybe oder of test file matters

**Comment by oliverbarnes** [comment]
:+1:

**Comment by oliverbarnes** [comment]
@Qianfinland just a tiny thing in CONTRIBUTING - pull request number and link, its description, and the issue it closes should go all on the same entry: https://github.com/oliverbarnes/participate-frontend/commit/0307dd7304f2006a1edaee9689ef1e0873a5d8fb

**Pull request merged**